packages/video_player/video_player/pubspec.yaml

```diff
@@ -33,6 +33,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  leak_tracker_flutter_testing: any
```

Comment:
I'm not sure what our version policies are for dev_deps. @stuartmorgan is there guidance on this?

Reply:
For more context, here is the documentation: https://prudential-ps.atlassian.net/browse/GTP-2844

> Put `any` instead of version, because the version is defined by your Flutter SDK.

Reply:
Yes, generally this would be a very bad idea, but it's specifically needed for the case of this package (IIUC because it's not in the SDK, so can't use `sdk:`, but is *pinned* by Flutter).